src/node_process_object.cc

```diff
@@ -105,18 +105,37 @@ MaybeLocal<Object> CreateProcessObject(Realm* realm) {
   READONLY_PROPERTY(
       process, "version", FIXED_ONE_BYTE_STRING(isolate, NODE_VERSION));
 
-  // process.versions
   Local<Object> versions = Object::New(isolate);
-  READONLY_PROPERTY(process, "versions", versions);
+  // Node.js version is always on the top
+  READONLY_STRING_PROPERTY(
+      versions, "node", per_process::metadata.versions.node);
+  std::vector<std::pair<std::string_view, std::string_view>> versions_vec;
 
 #define V(key)                                                                 \
-  if (!per_process::metadata.versions.key.empty()) {                           \
-    READONLY_STRING_PROPERTY(                                                  \
-        versions, #key, per_process::metadata.versions.key);                   \
-  }
+  do {                                                                         \
+    versions_vec.push_back(                                                    \
+        std::make_pair(#key, per_process::metadata.versions.key));             \
```

Comment:
This should fix the `process.versions` bug:
```suggestion
    versions_vec.emplace_back(                                                 \
        #key, per_process::metadata.versions.key);                             \
```
Looks like the issue you are facing is pretty much a variation of https://stackoverflow.com/q/65746198/11395352.

Reply:
Thanks a lot. I'm confused about CI error for the days